packages/toCapitals/src/index.js

```diff
@@ -0,0 +1,18 @@
+/**
+ * @description Converts the first character of the input text to upper case. If the second parameter is true, the method converts the rest of the subject to lower case.
+ * @param {String} text
+ * @param {Boolean} isLowercase
+ * @example
+ * toPred('foo Bar') // returns 'Foo Bar'
+ * toPred('fOO Bar') // returns 'FOO Bar'
+ * toPred('fOO Bar', true) // returns 'Foo bar'
+ */
+const toCapitals = (text, isLowercase = false) => {
```

Comment:
Instead of testing `text`, you can initialize it as:
`const toCapitals = (text = '', isLowercase = false) => {`

Reply:
thank you